src/actions/index.js

```diff
@@ -0,0 +1,82 @@
+import bookService from '../services/books-stub'
+import {
+  START_LOADER, STOP_LOADER, CREATE_BOOK,
+  SET_BOOKS_COLLECTION, REMOVE_BOOK, CHANGE_FILTER,
+  SHOW_ALERT, HIDE_ALERT
+} from './actionConstants'
+import { generateErrorText } from '../libs/helpers'
+
+const startLoading = () => ({ type: START_LOADER })
+const stopLoading = () => ({ type: STOP_LOADER })
+
+export const setBooks = (books) => ({
+  type: SET_BOOKS_COLLECTION,
+  books
+})
+
+export const createBook = (book) => ({
+  type: CREATE_BOOK,
+  book
+})
+
+export const removeBook = (book) => ({
```

Comment:
The project instructions did a bit of an overkill. We can simply pass `id` here in removing a book. So yeah I would recommend that:

```javascript
export const removeBook = id => ({
  type: REMOVE_BOOK,
  id
})
```